src/test/java/bio/terra/service/filedata/azure/AzureSynapsePdaoSnapshotConnectedTest.java

```diff
@@ -0,0 +1,443 @@
+package bio.terra.service.filedata.azure;
+
+import static bio.terra.common.PdaoConstant.PDAO_ROW_ID_COLUMN;
+import static bio.terra.common.PdaoConstant.PDAO_ROW_ID_TABLE;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import bio.terra.common.EmbeddedDatabaseTest;
+import bio.terra.common.Relationship;
+import bio.terra.common.SynapseUtils;
+import bio.terra.common.category.Connected;
+import bio.terra.common.exception.PdaoException;
+import bio.terra.common.fixtures.ConnectedOperations;
+import bio.terra.common.fixtures.JsonLoader;
+import bio.terra.model.*;
```

Comment:
Looks like it rolled up a couple of these imports

Reply:
I think I've fixed these! Thanks!